string name
int32 height
int32 width
